tests/features/asset_distribution/add_solution.feature

```diff
@@ -0,0 +1,53 @@
+@api
+Feature: "Add asset distribution" visibility options.
+  In order to manage distributions
+  As a moderator
+  I need to be able to add "Asset distribution" rdf entities through UI.
+
+  Scenario: "Add distribution" button should only be shown to moderators.
+    Given the following collection:
+      | name | Asset Distribution Test                  |
+      | uri  | https://a.distribution/solution/add/test |
+      | logo | logo.png                                 |
+    And the following solution:
+      | name              | Asset random name                        |
+      | uri               | http://joinup.eu/ad/solution/api/bar     |
+      | description       | Some reusable random description         |
+      | elibrary creation | 1                                        |
+      | groups audience   | https://a.distribution/solution/add/test |
```

Comment:
The functionality of the URI generation is occurring in the widget of the field. It is not a storage level functionality, nor the definition of field. This means that direct assignment of the URI is possible. Should we convert the behat to match the current UI? We can just remove it from the Acceptance criteria.